Source/WebCore/rendering/svg/SVGResources.cpp

```diff
@@ -232,7 +232,7 @@ std::unique_ptr<SVGResources> SVGResources::buildCachedResources(const RenderEle
             AtomString id(clipPath.fragment());
             if (auto* clipper = getRenderSVGResourceById<LegacyRenderSVGResourceClipper>(treeScope, id))
                 ensureResources(foundResources).setClipper(clipper);
-            else
+            else if (!renderer.document().settings().layerBasedSVGEngineEnabled())
```

Comment:
I guess this is temporary, since we don't want LBSE to use this SVGResources system.

Reply:
Yes, a quick fix for testing, we will need some/similar logic here for every LBSE resource we introduce.

Reply:
Great!